### ISSUE TITLE
Router link with click function

### ISSUE THREAD

**Issue by praveen4554** [angular/angular#17759]
I have a requirement in the requirement we want 2 tabs when I click the send tab we should display + icon on beside of the 2 and tab and also when I click on the second tab it should to generate a click function. When I click the first tab I should hide the + icon I don't know how to start the requirement for the first 2 routes implemented router link

Thanks,
Praveen

**Comment by Toxicable** [comment]
Hello, we reviewed this issue and determined that it doesn't fall into the bug report or feature request category. This issue tracker is not suitable for support requests, please repost your issue on [StackOverflow](http://stackoverflow.com/) using tag `angular`.

If you are wondering why we don't resolve support issues via the issue tracker, please [check out this explanation](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-got-a-question-or-problem).

**Comment by lanusha** [comment]
Yes I do have the same problem .some one help us out please.. Thanks in Advance..

attached a sample image for reference 
![git](https://user-images.githubusercontent.com/25409940/27521849-217c1836-59ef-11e7-834f-2bf09dd1a1ed.PNG)
